livekit/src/room/data_stream/outgoing.rs

```diff
@@ -74,7 +74,7 @@ impl<'a> StreamWriter<'a> for ByteStreamWriter {
         &self.info
     }
 
-    async fn write(&self, bytes: &[u8]) -> StreamResult<()> {
+    async fn write(&self, bytes: &'a [u8]) -> StreamResult<()> {
```

Comment:
the lifetime annotation here was needed to make it compile locally on my setup (v1.83) - not sure why this wasn't failing on CI, any drawbacks around adding this?